tools/fake-dbus-module/dbus/__init__.py

```diff
@@ -1,10 +1,8 @@
-import dbus.exceptions
```

Comment:
it should stay so users import dbus.exceptions from fake-dbus-module as in real dbus module, right?

Reply:
Wouldn't that import only apply inside that file, and it doesn't use any exceptions?

Reply:
This one is probably needed, since `dbus.exceptions.DBusException` is used in app.py without any `have_dbus`-style guards.